src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralConcluderTest.kt

```diff
@@ -190,11 +190,11 @@ internal class ReferralConcluderTest {
   }
 
   @Test
-  fun `should not flag end of service report as required when no sessions have been attempted`() {
```

Comment:
should we keep 1 test case for regression that merely an "attempted" session will **not** count towards 'delivery' status? (and update the below maps)

<img width="453" alt="image" src="https://user-images.githubusercontent.com/1526295/165346515-f3b4ef5f-444b-41d4-b570-a32609a3ea5e.png">

Reply:
We aren't calling that repository method in this flow anymore. So we can't really still test it in that way? We have a test that returns '0' from the repository, simulating that no sessions were attended if that's what you mean?

Reply:
Was thinking of explicitly testing the "1 delivery session with 'no' attendance -> must not flag EoSR as required" -- but I might be missing something

Reply:
Perhaps a resolution could be then to add a test for `countNumberOfAttendedSessions`?

Reply:
added some repository tests to ensure that only the attended values (YES/LATE) are included in the count. As well as scheduled sessions with no attendance not being included in the count